.github/workflows/dependabot-auto-evaulate-pr.yaml

```
@@ -0,0 +1,31 @@
+name: Dependabot Evaluation
+
+on:
+    pull_request:
+        paths:
+            - "cookbook/**/*"
+
+jobs:
+    evaluate_dependabot_pr:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v2
+
+            - name: Set up Python
+              uses: actions/setup-python@v2
+              with:
+                  python-version: 3.x
```

just to be explicit that we're 3.9+
```suggestion
                  python-version: 3.11
```